ocfweb/docs/templates/docs/partials/account.md

```diff
@@ -0,0 +1,37 @@
+* Rather than iTunes/QuickTime or Windows Media Player, users can access music or video files using VLC.
+* The pre-installed text editor is Atom, and the pre-installed IDE is Visual Studio Code.
+* The OCF does not currently provide a true replacement for Adobe Acrobat; PDFs can be opened with the Document Viewer and edited using other applications.
+Most of these applications can be found on the desktop, and all applications can be accessed from the menu in the top-left corner of the screen, analogous to the Start menu in Windows.
+
+## Persistent File Storage
+
+When you log out of one of our desktop computers, any files you saved to that computer will be erased. This is because most files saved to these computers are immediately printed, so there is no need to take up disk space holding onto them. However, some users may wish to have access to their files every time they log in to an OCF desktop. As part of your OCF account, you gain access to 5 GB of free file storage on OCF servers. This file storage is accessible by clicking on the "OCF File Storage" icon on any desktop computer or by SSHing into our account server (tsunami.ocf.berkeley.edu, and if you don't know what SSH means, read [our guide to the shell](LINK-GOES-HERE)).
+
+Files placed in the "OCF File Storage" folder are accessible from any desktop computer and are saved even after you log out, whereas files placed in the "Home" folder are local to that computer and are deleted upon log-out. The "OCF File Storage" folder should contain a folder called "public_html", which is used for web hosting – see below.
+
+## Free Printing
+
+Of course, printing is central to many users' experiences with the OCF. By signing up for an account, you get 200 free sides of printing (200 pages single-sided, 100 pages double-sided). We print black-and-white only, in economode (meaning the ink may be a bit lighter than expected). Users are limited to 20 sides of printing per day. We ask that users download or export any documents they wish to print as PDFs first, as these work best with our print server.
+
+## Web Hosting
+
+When you create an OCF account, you also create a subdomain on our website: ocf.berkeley.edu/\~yourusername. You can create your own website using HTML/CSS/JS or use a system like WordPress or Django. Alternatively, you can use your website as file storage. Either way, your website is sourced directly from the aforementioned "public_html" folder – meaning this is the folder in which you should place all content for your website. More technical information can be found at [our web hosting page](https://www.ocf.berkeley.edu/docs/services/web/).
+
+If you are a signatory of a student group and would like to request a \*.berkeley.edu website, you will need to create a group account and complete a few steps before making your request. Instructions can be found on [our vhost page](https://www.ocf.berkeley.edu/docs/services/vhost/).
```

Comment:
A lot of users who really want group accounts might accidentally come to this page, looking for this informatoin. Instead of hiding this information in the middle of the page, mention this distinction at the beginning and link to a separate page for group accounts.